packages/MarkdownEditor-Tests.package/WeakSmalltalkParserTest.class/instance/testUnkownLowecaseSymbolIsInstanceVariable.st

```diff
@@ -0,0 +1,9 @@
+tests
+testUnkownLowecaseSymbolIsInstanceVariable
```

Comment:
should be 'LowercaseSymbol'
And maybe we can find a better word for symbol (or you can convince me to keep it that way), but the parser doesn't think in symbols, actually they use the word 'token' in their code, or they start comparing 'strings'. Maybe even something like 'unknownVariableIsTreatedLikeInstanceVariable' or something.

Reply:
Often they use keyword. Would this be okay, too?

Reply:
Yep, that sounds good. But I'm still kinda stumbling over the 'isInstanceVariable'. Because we don't actually care whether it is one or not, we just want it looking like one. Haha, maybe 'looksLikeInstanceVariable' xD No I don't know, but do you get what I mean?

Reply:
It is parsed as such. This is what we check. We don't check whether it looks like it.